docs/products/kafka/howto/enable-kafka-tiered-storage.rst

```diff
@@ -0,0 +1,92 @@
+Enable tiered storage for Aiven for Apache Kafka®
+=====================================================
+Learn how to enable tiered storage capability of Aiven for Apache Kafka®. This topic provides step-by-step instructions for maximizing storage efficiency using either the `Aiven console <https://console.aiven.io/>`_ or the :doc:`Aiven CLI </docs/tools/cli>`.
+
+.. important:: 
+    
+   Aiven for Apache Kafka® tiered storage is an early availability feature, which means it has some restrictions on the functionality and service level agreement. It is intended for non-production environments, but you can test it with production-like workloads to assess the performance. To enable this feature, navigate to the :doc:`Feature preview </docs/platform/howto/feature-preview>` page within your user profile.
+
+Prerequisites
+--------------
+* Aiven account and a project set up in the Aiven Console
+* Aiven for Apache Kafka® service with Apache Kafka version 3.6
+* Aiven CLI
+
+
+Enable tiered storage via Aiven Console
+------------------------------------------
+Follow these steps to enable tiered storage for your service using the Aiven Console. 
+
+1. Access the  `Aiven console <https://console.aiven.io/>`_, and select your project.
+2. Create a new :doc:`Aiven for Apache Kafka service </docs/platform/howto/create_new_service>` or choose an existing one.  
+
+   - If you are creating a new service:
+
+     a. On the **Create Apache Kafka® service** page, scroll down to the **Tiered storage** section.
+     b. To enable tiered storage, select the **Enable tiered storage** toggle.
+   
+   - If you are using an existing service:
+
+     a. Go to the service's **Overview** page, scroll down to the **Tiered storage** section.
+     b. To enable tiered storage, select the **Enable tiered storage** toggle.
+
+3. Enter the value for **Local cache size (bytes)**. This value indicates the amount of memory reserved for storing frequently accessed data, enhancing the speed of reading data from remote storage.
+
+4. Select the **Activate tiered storage** to save your settings and enable tiered storage for the service. 
+
+
+Configuring default retention policies at service-level
```

Comment:
Should we mention the other place? The tiered storage view.

Reply:
Sure, will add this.